Art/Event/rescued_saplings_comfy_sleepy.png.import

```diff
@@ -0,0 +1,34 @@
+[remap]
```

Comment:
the persona should be in NPC
also they don't seem to respect the naming convention